docs/en/observability/tab-widgets/filebeat-logs/content.asciidoc

```diff
@@ -0,0 +1,75 @@
+// tag::logs[]
+
+. Follow the {filebeat-ref}/filebeat-installation-configuration.html[Filebeat quick start] to learn how to
+install {filebeat}, connect to the {stack}.
+
+. Add the following configuration to your `filebeat.yaml` file to start collecting log data.
++
+**structured logs**
```

Comment:
[question] do you think that we could have completely separate snippets for plaintext and ECS logs ? having the two in the same place means a part of it won't be relevant depending on where we insert the widget.

Reply:
Yes! I was planning on separating this out as, for example, the unstructured section doesn't make sense on the ECS reformatting page. I'll make that change.

Reply:
Fixed in https://github.com/elastic/observability-docs/pull/2452/commits/ef578dbcbb599be0dc52accf37161e4f587f113f.